shelley-ma/formal-spec/utxo.tex

```diff
@@ -226,18 +226,8 @@ \subsection*{Witnessing}
 \end{figure}
 
 Figure~\ref{fig:functions-witnesses} contains the changed definition
-of the function $\fun{scriptsNeeded}$, to also collect the scripts
-necessary for forging.
-
-Recall here that the $\Script$ type has changed to include
-not just multi-signature script type $\ScriptMSig$, but also another native (i.e. evaluated
-by the ledger directly) script type, $\ScriptMPS$. The witnessing rule UTXOW
-does not itself change, but the function $\fun{validateScript}$, which it calls
-to evaluate the scripts prescribed by $\fun{scriptsNeeded}$, does change, including
-a change to the arguments passed to it, see~\ref{sec:mps-lang}.
-
-Note that, same as in Shelley, the validation function called by the
-UTXOW rule, which checks a given script, does not know the purpose of the script
-(forging, output-locking, etc.).
-All scripts are run in the same way, with $\fun{validateScript}$ being responsible
-for calling the evaluator appropriate for the specific type of script.
+of the function $\fun{scriptsNeeded}$, which also collects the scripts
+necessary for minting.
+
+The witnessing rule UTXOW only needs a minor change for verifying the meta data hash,
+which is replacing the condition $\var{md} = \Nothing$ with $\var{md} = (\emptyset, \Nothing)$.
```

Comment:
i think it would be good to write the full check on a new line, so that it is easier to spot the change.

Reply:
I'm not sure what you mean, this is a single sentence that has its own paragraph. I wrote it like this because it's a really minor change that I don't think requires repeating the whole UTXOW rule.